src/openms/include/OpenMS/METADATA/SpectrumMetaDataLookup.h

```diff
@@ -283,6 +298,8 @@ namespace OpenMS
     /// Assignment operator (not implemented)
     SpectrumMetaDataLookup& operator=(const SpectrumMetaDataLookup&);
 
+    /// adds Ion matches to identifcations
+    static void addIonMatches_(PeptideIdentification& pi, const MSSpectrum<Peak1D> &spec);
```

Comment:
This information might well be present from the search engine results as spectra have been potentially already been annotated using custom algorithms. But we might not carry over the information.
It is cleary no lookup as no existing information is looked up but recalculated.